ledger/sandbox-classic/src/main/scala/platform/sandbox/stores/LedgerBackedWriteService.scala

```diff
@@ -35,6 +35,7 @@ private[stores] final class LedgerBackedWriteService(ledger: Ledger, timeProvide
       submitterInfo: SubmitterInfo,
       transactionMeta: TransactionMeta,
       transaction: SubmittedTransaction,
+      estimatedInterpretationCost: Long,
```

Comment:
So we don't actually use this?

Reply:
Not here. We use it in e.g. `KeyValueParticipantState` or `InterpretationCostBasedLedgerWriterChooser`.